.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Thank you for contributing to Amplication :)
+
+PLEASE, GO THROUGH THESE STEPS BEFORE YOU SUBMIT A PR!
+
+Make sure that:
+
+1. There is an open issue for this PR. If not, please open one before submitting your changes. Before proceeding, any change needs to be discussed (You can skip this if you're fixing a typo or adding an app to the Showcase).
+
+2. You have done your changes in a separate branch. Branches MUST have descriptive names that start with either the `fix/[issue #]-` or `feature/[issue #]-` prefixes. Good examples are: `fix/404-signin-issue` or `feature/201-new-templates`.
+
+3. Your pull request MUST NOT target this repository's `master` branch. You probably want to target a branch name that starts with `release/` instead.
+
+4. You give a descriptive title to your PR.
```

Comment:
```suggestion
4. You gave a descriptive title to your PR.
```

Reply:
maybe `You are giving a descriptive title to your PR.` will be better?

Reply:
Agreed